dashing/distribution.yaml

```diff
@@ -2109,6 +2109,24 @@ repositories:
       url: https://github.com/ros-visualization/qt_gui_core.git
       version: crystal-devel
     status: maintained
+  raptor_dbw_ros2:
+    release:
+      packages:
+      - raptor_can_dbc_parser
+      - raptor_dbw_can
+      - raptor_dbw_joystick
+      - raptor_dbw_msgs
+      - raptor_pdu
+      - raptor_pdu_msgs
+      tags:
+        release: release/dashing/{package}/{version}
+      url: https://github.com/NewEagleRaptor/raptor-dbw-ros2-release.git
+      version: 1.0.0-1
+    source:
+      type: git
+      url: https://github.com/NewEagleRaptor/raptor-dbw-ros2.git
+      version: 1.0.0-dashing
```

Comment:
This should point to a branch, not a tag.

Reply:
Fixed (I'm helping out NewEagle).

Reply:
Yep, we appreciate it